src/components/connect.tsx

```diff
@@ -60,7 +60,9 @@ export type UrqlProps<Data, Mutations = {}> = {
   refreshAllFromCache: () => void;
 } & Mutations;
 
-export default class Connect extends Component<IConnectProps> {
+export default class Connect<Data = {}, Mutations = {}> extends Component<
```

Comment:
This is how the TS team suggested I do the default in Formik. It improves the error stack when TS users forget to pass `UrqlProps` a generic for their data.